src/commands/appdistribution-distribute.ts

```diff
@@ -127,7 +108,7 @@ module.exports = new Command("appdistribution:distribute <release-binary-file>")
     utils.logBullet("uploading binary...");
     let releaseName;
     try {
-      const operationName = await requests.uploadRelease(distribution);
+      const operationName = await requests.uploadRelease(appId,distribution);
```

Comment:
Should be a space before `distribution` i.e. `requests.uploadRelease(appId, distribution)`